src/Shell/Task/ExtractTask.php

```diff
@@ -122,7 +122,8 @@ protected function _getPaths()
             $response = $this->in($message, null, $defaultPath);
             if (strtoupper($response) === 'Q') {
                 $this->err('Extract Aborted');
-                return $this->_stop();
+                $this->_stop();
+                return;
```

Comment:
I did not remember we had those stray calls there. Actually both should be removed and put nowhere, I think they are not needed.